src/server/server.js

```diff
@@ -1,22 +1,34 @@
 import {SocketCluster} from 'socketcluster';
-import {auth0Secret} from './secrets';
 import path from 'path';
+import dotenv from 'dotenv';
+import dotenvExpand from 'dotenv-expand';
 // import os from 'os';
 
+// Import .env and expand variables:
+try {
```

Comment:
let's move this to a function. try/catch can't be optimized by the compiler so whatever function has it is guaranteed to be slow